composer.json

```diff
@@ -8,35 +8,26 @@
             "email": "info@palantir.net"
         }
     ],
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "config": {
-        "platform": {
-            "php": "7.2"
-        },
-        "preferred-install": {
-            "*": "dist"
-        },
-        "sort-packages": true
-    },
     "repositories": [
         {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         }
     ],
     "require": {
-        "composer/installers": "^1.0",
-        "drupal/core-composer-scaffold": "^8.8",
+        "composer/installers": "^1.2",
+        "drupal/coder": "^8.3.6",
         "drupal/config_installer": "^1.8",
         "drupal/config_split": "^1.4",
-        "drupal/core-recommended": "^8.7.0",
+        "drupal/core-composer-scaffold": "^8.8",
+        "drupal/core-project-message": "^8.8",
+        "drupal/core-recommended": "^8.8",
         "zaporylie/composer-drupal-optimizations": "^1.1"
     },
     "require-dev": {
-        "drupal/core-dev": "^8.7.0",
+        "drupal/core-dev": "^8.8",
         "drupal/drupal-extension": "^3.1",
-        "palantirnet/the-build": "^2.0",
+        "palantirnet/the-build": "dev-8.8-fine-tuning",
```

Comment:
I won't merge this PR until it's on the release version of the-build :)

Reply:
:+1: